umbra/umbra.c

```diff
@@ -758,3 +758,35 @@ umbra_get_shared_shadow_block(IN  umbra_map_t *map,
         return DRMF_ERROR_INVALID_PARAMETER;
     return umbra_get_shared_shadow_block_arch(map, value, value_size, block);
 }
+
+DR_EXPORT
+drmf_status_t umbra_get_granularity(const umbra_map_t *map, OUT int *scale,
+                                    OUT bool *is_scale_down) {
+
+  if (map == NULL || scale == NULL || is_scale_down == NULL)
+    return DRMF_ERROR_INVALID_PARAMETER;
```

Comment:
style: every line in the function: indent 4

(I do want to get clang-format here; have not gotten to it yet)

Reply:
Done.